src/dashboard/package.json

```diff
@@ -43,6 +43,7 @@
     "eslint": "^8.55.0",
     "eslint-config-next": "14.0.4",
     "postcss": "^8.4.32",
+    "sass": "^1.69.5",
```

Comment:
Documentation recommended installing this if we're using sass, but so far I can't see much difference.